kubeconfig/stage_managed_release_kubelogin

```diff
@@ -25,5 +25,4 @@ users:
       command: kubectl
       env: null
       interactiveMode: IfAvailable
-      provideClusterInfo: false
-      
+      provideClusterInfo: false
```

Comment:
It seems newline is missing here? At least I assume that's what this "wrong way" sign means.